torchbenchmark/models/hf_MPT_7b_instruct/__init__.py

```diff
@@ -0,0 +1,14 @@
+from torchbenchmark.tasks import NLP
+from torchbenchmark.util.framework.huggingface.model_factory import HuggingFaceModel
+
+class Model(HuggingFaceModel):
+    task = NLP.LANGUAGE_MODELING
+    # https://huggingface.co/mosaicml/mpt-7b
+    DEFAULT_TRAIN_BSIZE = 4
+    DEFAULT_EVAL_BSIZE = 1
+
+    def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
```

Comment:
Similarly, can you remove the `jit=False` argument and the `jit=jit` argument below? Sorry I missed this in my previous review.